Source/controls/plrctrls.cpp

```diff
@@ -1496,7 +1528,7 @@ void PerformPrimaryAction()
 		if (pcurs > CURSOR_HAND && pcurs < CURSOR_FIRSTITEM) {
 			TryIconCurs();
 			NewCursor(CURSOR_HAND);
-		} else {
+		} else if (pcursinvitem != -1) {

```

Comment:
It seems this change makes it impossible to place items in the inventory using gamepad.

Reply:
Oops. Fixed :)